src/promotion/promotion.h

```diff
@@ -44,26 +62,28 @@ ocf_error_t ocf_promotion_set_policy(ocf_promotion_policy_t policy,
 /**
  * @brief Set promotion policy parameter
  *
- * @param[in] policy promotion policy handle
+ * @param[in] cache cache handle
  * @param[in] param_id id of parameter to be set
+ * @param[in] type id of promotion policy to be configured
  * @param[in] param_value value of parameter to be set
  *
  * @retval ocf_error_t
  */
-ocf_error_t ocf_promotion_set_param(ocf_promotion_policy_t policy,
-		uint8_t param_id, uint32_t param_value);
+ocf_error_t ocf_promotion_set_param(ocf_cache_t cache, uint8_t param_id,
+		ocf_promotion_t type, uint32_t param_value);
 
 /**
  * @brief Get promotion policy parameter
  *
- * @param[in] policy promotion policy handle
+ * @param[in] cache cache handle
  * @param[in] param_id id of parameter to be set
+ * @param[in] type id of promotion policy to be configured
  * @param[out] param_value value of parameter to be set
  *
  * @retval ocf_error_t
  */
-ocf_error_t ocf_promotion_get_param(ocf_promotion_policy_t policy,
```

Comment:
Please reorder the parameters to have promotion type as the second arg - this would make more sense and it would be consistent with current convention (cleaning params)